submissions/akulahere/html-css-popup/css/main.css

```diff
@@ -0,0 +1,138 @@
+body {
+    font-family: "Times New Roman";
+    margin: 0px;
+}
+
+a {
+    text-decoration: none;
+    color:black;
```

Comment:
I recommend add padding for better visibility.

Reply:
fixed